model-optimizer/extensions/front/tf/CTCGreedyDecoderReplacement.py

```diff
@@ -67,38 +71,25 @@ def replace_sub_graph(self, graph: Graph, match: dict):
         # TODO: Once Inference Engine's CTCGreedyDecoder starts to support sequence length format like in TensorFlow,
         # CTCGreedyDecoderReplacement2 needs to be removed and CTCGreedyDecoderReplacement, a more generic
         # transformation, needs to be adopted for all cases
```

Comment:
please remove this comment and per my understanding `CTCGreedyDecoderReplacement2` must be removed.

Reply:
this is also unresolved.